app/src/main/java/org/mozilla/focus/tiles/HomeTilesManager.kt

```diff
@@ -7,13 +7,90 @@ package org.mozilla.focus.tiles
 import android.content.Context
 import android.content.Context.MODE_PRIVATE
 import android.content.SharedPreferences
+import android.graphics.BitmapFactory
 import android.support.annotation.UiThread
 import org.json.JSONArray
+import org.json.JSONObject
+import org.mozilla.focus.utils.UrlUtils
+import java.net.URL
 import java.util.UUID
 
 private const val PREF_HOME_TILES = "homeTiles"
 private const val CUSTOM_SITES_LIST = "customSitesList"
 
+private const val BUNDLED_HOME_TILES_DIR = "bundled/"
+private const val HOME_TILES_JSON_PATH = BUNDLED_HOME_TILES_DIR + "bundled_tiles.json"
+private const val HOME_TILES_JSON_KEY = "bundled_tiles"
+
+class BundledTilesManager private constructor(context: Context) {
```

Comment:
Can we add a comment here (or where the JSON is loaded) explaining how we expect the URLs to be exact matches as much as possible?